README.zh-cn.md

```diff
@@ -0,0 +1,344 @@
+Username:
+Email:
+Password:
+```
+
+在 Activeloop Cloud 中初始化一个空数据集：
+
+```python
+import hub
+
+ds = hub.empty("hub://<USERNAME>/test-dataset")
+```
+
+
+接下来，创建一个张量来保存我们刚刚初始化的数据集中的图像：
+
+```python
+images = ds.create_tensor("images", htype="image", sample_compression="jpg")
+```
+
+假设您有一个图像文件路径列表，让我们将它们上传到数据集：
+
+```python
+image_paths = ...
+with ds:
+    for image_path in image_paths:
+        image = hub.read(image_path)
+        ds.images.append(image)
+```
+
+或者，您也可以上传 numpy 数组。由于 `images` 张量是使用 `sample_compression="jpg"` 创建的，数组将使用 jpeg 压缩进行压缩。
+
+```python
+import numpy as np
+
+with ds:
+    for _ in range(1000):  # 1000 random images
+        random_image = np.random.randint(0, 256, (100, 100, 3))  # 100x100 image with 3 channels
+        ds.images.append(random_image)
+```
+
+
+
+### 🚀 如何加载 Hub 数据集
+
+
+您可以使用一行代码加载刚刚创建的数据集：
+
+```python
+import hub
+
+ds = hub.load("hub://<USERNAME>/test-dataset")
+```
+
+您还可以访问 Hub 格式的<a href="https://docs.activeloop.ai/datasets/?utm_source=github&utm_medium=github&utm_campaign=github_readme&utm_id=readme"> 100 多个图像、视频和音频数据集</a>，而不仅仅是您创建的那些。以下是加载 [Objectron Bikes 数据集](https://github.com/google-research-datasets/Objectron)的方法:
+
+```python
+import hub
+
+ds = hub.load('hub://activeloop/objectron_bike_train')
+```
+
+要以 numpy 格式获取 Objectron Bikes 数据集中的第一张图像：
+
+
+```python
+image_arr = ds.image[0].numpy()
+```
+
+
+
+## 📚 文档
+可以在我们的[文档页面](http://docs.activeloop.ai/?utm_source=github&utm_medium=repo&utm_campaign=readme)上找到入门指南、示例、教程、API 参考和其他有用信息。
+
+## 🎓 对于学生和教育工作者
+Hub 用户可以通过与 Activeloop 平台的免费集成来访问和可视化各种流行的数据集。用户还可以创建和存储自己的数据集，并将其提供给公众。高达 300 GB 的免费存储空间可供学生和教育工作者使用：
+
+| <!-- -->                          | <!-- -->   |
+|-----------------------------------|------------|
+| Activeloop 托管的公共数据集的存储 | 200GB 免费 |
+| Activeloop 托管的私有数据集的存储 | 100GB 免费 |
+
+
+
+## 👩‍💻 与熟悉工具的比较
+
+
+<details>
+  <summary><b>Activeloop Hub vs DVC</b></summary>
+  
+Hub 和 DVC 为数据提供类似于 git 的数据集版本控制，但它们存储数据的方法有很大不同。Hub 将数据转换并存储为分块压缩数组，从而可以快速流式传输到 ML 模型，而 DVC 在存储在效率较低的传统文件结构中的数据之上运行。当数据集由许多文件（即许多图像）组成时，与 DVC 的传统文件结构相比，Hub 格式使数据集版本控制更加容易。另一个区别是 DVC 主要使用命令行界面，而 Hub 是 Python 包。最后，Hub 提供了一个 API，可以轻松地将数据集连接到 ML 框架和其他常见的 ML 工具，并通过[Activeloop 的可视化工具](http://app.activeloop.ai/?utm_source=github&utm_medium=repo&utm_campaign=readme)实现即时数据集可视化.
+</details>
+
+
+<details>
+  <summary><b>Activeloop Hub vs TensorFlow 数据集 (TFDS)</b></summary>
+  
+Hub 和 TFDS 将流行的数据集无缝连接到 ML 框架。Hub 数据集与 PyTorch 和 TensorFlow 兼容，而 TFDS 仅与 TensorFlow 兼容。Hub 和 TFDS 之间的一个关键区别在于，Hub 数据集是为从云端流式传输而设计的，而 TFDS 必须在使用前在本地下载。因此使用 Hub 可以直接从 TensorFlow 数据集导入数据集，并将它们流式传输到 PyTorch 或 TensorFlow。除了提供对流行的公开数据集的访问之外，Hub 还提供强大的工具来创建自定义数据集，将它们存储在各种云存储提供商上，并通过简单的 API 与他人协作。TFDS 主要专注于让公众轻松访问常用数据集，而自定义数据集的管理不是主要关注点。一个详细的对比介绍可以在[这里](https://www.activeloop.ai/resources/7jWZXOEJwDoNJS25uiforF/tensorflow-tf.data-&-hub:-how-to-implement-your-tensorflow-data-pipelines-with-hub-/?utm_source=github&utm_medium=repo&utm_campaign=readme)看到.
```

Comment:
also the correct link is (https://www.activeloop.ai/resources/tensor-flow-tf-data-activeloop-hub-how-to-implement-your-tensor-flow-data-pipelines-with-hub/)

Reply:
thank you for updating this